src/amber/cli/templates/error/controller/src/controllers/error_controller.cr.ecr

```diff
@@ -0,0 +1,15 @@
+class ErrorController < Amber::Controller::Error
+  LAYOUT = "application.<%= @language %>"
+
+  def forbidden
+    render("forbidden.<%= @language %>")
+  end
+
+  def not_found
+    render("not_found.<%= @language %>")
+  end
+
+  def internal_server_error
+    render("internal_server_error.<%= @language %>")
+  end
+end
```

Comment:
I went back and forth between using that status codes as file names and this - any thoughts?

Reply:
Yeah, I think status codes are too implicit, error name is more readable

Reply:
I prefer the names as well

Reply:
maybe we should loop thru these names to tidy up, WDYT?